tpot/base.py

```diff
@@ -94,6 +93,25 @@ def handler(dwCtrlType, hook_sigint=_thread.interrupt_main):
 
     win32api.SetConsoleCtrlHandler(handler, 1)
 
+def isnotebook():
```

Comment:
Suggestion: Refactor to `is_notebook()`.

Reply:
OK